publisher/providers/pyth_replicator.py

```diff
@@ -86,3 +122,24 @@ def latest_price(self, symbol: Symbol) -> Optional[Price]:
             return None
 
         return Price(price, conf)
+
+
+def manual_aggregate(prices: List[float]) -> Tuple[float, float]:
+    """
+    This function is used to manually aggregate the prices of the active publishers. This is a very simple
+    implementation that does not get the median and confidence accurately but it is good enough for our use case.
+    """
+    prices.sort()
+    no_prices = len(prices)
+
+    # Ensure the price never goes below zero
+    agg_price = max(0, prices[no_prices // 2])
+
+    agg_confidence_interval_left = agg_price - prices[no_prices // 4]
+    agg_confidence_interval_right = prices[no_prices * 3 // 4] - agg_price
+
+    # Make sure agg_price - agg_confidence interval is never negative
+    agg_confidence_interval = min(
+        agg_price, max(agg_confidence_interval_left, agg_confidence_interval_right, 0)
```

Comment:
Im pretty sure `agg_confidence_interval_left` and `agg_confidence_interval_right` are always positive or 0